tests/test_cli.py

```diff
@@ -139,3 +189,62 @@ def test_triton_profile(self, mocker, monkeypatch):
         args = parse_args()
         args.func(args)
         mock_run.assert_called_once_with(["genai-perf", "-m", "add_sub"], check=True)
+
+    @pytest.mark.parametrize("model", ["mock_llm"])
```

Comment:
Let's throw `add_sub` in here too to be consistent with the other tests.

Reply:
Removed `add_sub` from `test_triton_config()` and `test_triton_status()`